### PR TITLE
fix stateful operator & fix from_df

### PR DESCRIPTION
**Pull request by reiase**
Signed-off-by: reiase <reiase@gmail.com>

### CODE REVIEW

File: tests/unittests/functional/test_entity.py [hunk 1]
```diff
@@ -22,7 +22,7 @@ class TestEntity(unittest.TestCase):
     def test_init(self):
         d = {'a': 'A', 'b': 'B', 'c': 'C'}
         e1 = Entity()
-        e2 = Entity(d)
+        e2 = Entity(**d)
```
Comment (Chiiizzzy) [inline tests/unittests/functional/test_entity.py:25]: Entity class support loading data from a dict directly.

Comment (reiase) [reply tests/unittests/functional/test_entity.py:25]: I'm facing a problem: the argument name `target` may conflict with the keywords argument when converting a pandas data frame to entities.

File: towhee/operator/stateful_operator.py [hunk 2]
```diff
@@ -52,6 +52,7 @@ def __init__(self, name):
         super().__init__()
```
Comment (Chiiizzzy) [inline towhee/operator/stateful_operator.py:52]: We might need to make StatefulOperator a subclass of Operator

**Pull request merged**